bot/cogs/modlog.py

```diff
@@ -104,17 +104,34 @@ def ignore(self, event: Event, *items: int):
                 self._ignored[event].append(item)
 
     async def send_log_message(
-            self, icon_url: Optional[str], colour: Colour, title: Optional[str], text: str,
-            thumbnail: str = None, channel_id: int = Channels.modlog, ping_everyone: bool = False,
-            files: List[File] = None, content: str = None, additional_embeds: List[Embed] = None,
+            self,
+            icon_url: Optional[str],
+            colour: Colour,
+            title: Optional[str],
+            text: str,
+            thumbnail: str = None,
```

Comment:
Shouldn't the typehint for these (and the other `None` values) be `Optional[str]`?